awx/ui_next/src/components/DataListToolbar/DataListToolbar.jsx

```diff
@@ -93,9 +93,11 @@ function DataListToolbar({
               onRemove={onRemove}
             />
           </ToolbarItem>
-          <ToolbarItem>
-            <Sort qsConfig={qsConfig} columns={sortColumns} onSort={onSort} />
-          </ToolbarItem>
+          {sortColumns && (
```

Comment:
it's nice that we've included this as a fallback until we convert everything and remove sortColumns across the app.